crates/router/src/types/api/refunds.rs

```diff
@@ -1,6 +1,6 @@
 pub use api_models::refunds::{
-    RefundRequest, RefundResponse, RefundStatus, RefundType, RefundUpdateRequest,
-    RefundsRetrieveRequest,
+    RefundAggregateResponse, RefundRequest, RefundResponse, RefundStatus, RefundType,
```

Comment:
Why this import?

Reply:
using RefundAggregateResponse in core as `api::RefundAggregateResponse`, updated it to `api_models::refunds::RefundAggregateResponse`